common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -264,11 +264,15 @@ def create_response_element(self, **kwargs):
         *expect*: The value passed to the function cfn
 
         *answer*: Inline script that calculates the answer
+
+        *answer_attr*: The "answer" attribute on the tag itself (treated as an
```

Comment:
@amir-qayyum-khan This is actually part of the test =) The `CustomResponseXMLFactory` class I added it to is just a utility class used to generate custom response problems programatically for testing; it's not actually used in the platform itself.

The additional `answer_attr` parameter was necessary because the factory had no prior way to set the `answer` attribute on the generated problem (the existing `answer` parameter was for setting the inline answer *script*), which meant I couldn't write tests for it. (In reality it would probably be fine to omit it since it's just an alias for `expect`, but I figure the additional test coverage doesn't hurt... `answer` wasn't previously being tested anywhere.)